pages/README.md

```diff
@@ -17,3 +17,35 @@ const x = 'bla';
 ![Screenshot](_media/screenshot.png)
 
 [[News]](news/)
+
+## Abbreviations
+
+This is HTML abbreviation example.
+
+*[HTML]: Hyper Text Markup Language
+
+## Definition lists
+
+Term 1
+: Definition 1
+
+Term 2
+: Definition 2
+
+## Keyboard
+
+This is the [[help command]]
+
+## Element attributes
+
+dummy paragraph {data-toggle=modal}
+
+you should *style me*{.red} in red
+
+```python {data=asdf}
+nums = [x for x in range(10)]
+```
+
+## Sample (currently non fully working)
```

Comment:
What's with that?

Reply:
@jmuheim 
No idea why is not properly working.
I directly took it from https://www.npmjs.com/package/markdown-it-samp